improver_tests/acceptance/SHA256SUMS

```diff
@@ -633,73 +633,73 @@ e653ae271dd789dde8b03e6237cc7b48552dd9d5206d3430ee68d283683247a0  ./wind_downsca
-cc20c480460357559bbe9151ad069e97535df17a0687ae9e0fa1f5ef3cf9f9a6  ./wxcode-modal/spot_input/20201209T1100Z-weather_symbols-PT01H.nc
-950bee63559cf13b6af9f4b56d8d7a628a6afc77353345d44ea6798015c2148f  ./wxcode-modal/spot_input/20201209T1200Z-weather_symbols-PT01H.nc
-b479b8d6e9b002e716575a8a9051a70891f4cc8862af8ee80b53ecb05d37e8a3  ./wxcode-modal/spot_input/20201209T1300Z-weather_symbols-PT01H.nc
-2b1241b79c97e991ed06c34e6109e85dc1517726d2d375f104a7ac85d939d0a2  ./wxcode-modal/spot_input/20201209T1400Z-weather_symbols-PT01H.nc
-e42830916b5bf8b3f97ace0e54dec8e80be3cf93d9faf420274bab1cfb1772e4  ./wxcode-modal/spot_input/20201209T1500Z-weather_symbols-PT01H.nc
-b4bba902e61db128924d33f145e73aecb6cdf27dc71db6d66d17c9dcfcc498b5  ./wxcode-modal/spot_input/20201209T1600Z-weather_symbols-PT01H.nc
-09da6234ded5f1572183b8e297aa8f633fc20fe3571b5fc3d22b3a8c95c80a8a  ./wxcode-modal/spot_input/20201209T1700Z-weather_symbols-PT01H.nc
-56a8d8f763072027c4a4819e0cab298ea0b820cf32903ae010132fa0a72a2eb9  ./wxcode-modal/spot_input/20201209T1800Z-weather_symbols-PT01H.nc
-d081b4d643e2e7734a6a2084efd69a2a0a01ce90f8c234190729a4ceeba53c2b  ./wxcode-modal/spot_input/kgo.nc
-c68ee173a914f9dd5c47d7a528db0c6335aabe8b1506095f009f7f2b308f11b1  ./wxcode-modal/spot_ties/20201209T0700Z-weather_symbols-PT01H.nc
-5443a2d3502838bb83cc030f0e55867465dc4587a8407c9279204d0fcde7a27b  ./wxcode-modal/spot_ties/20201209T0800Z-weather_symbols-PT01H.nc
-35ff246048347536289053839c02b64255415aabe6763ca2b2289777b083a6ee  ./wxcode-modal/spot_ties/20201209T0900Z-weather_symbols-PT01H.nc
-1025491e3a2f207a2c1fd68d8a494bf54b9d0a4838bf35d54039eaeb0ab11778  ./wxcode-modal/spot_ties/20201209T1000Z-weather_symbols-PT01H.nc
-2e6d8a6ed2b7e4ae88c74b25ad04dc38d7cce58574a0c52339bf3076fe0b8929  ./wxcode-modal/spot_ties/20201209T1100Z-weather_symbols-PT01H.nc
-5bfc0319d35a7b7a230fdd51f3e8b6bfddf5632b32dde8e0ff023a6b106268f7  ./wxcode-modal/spot_ties/20201209T1200Z-weather_symbols-PT01H.nc
-8f94e92886c8df8070db44b273037b032615e4b8768a3e45c708686061f0c8bc  ./wxcode-modal/spot_ties/20201209T1300Z-weather_symbols-PT01H.nc
-4fb839e28d711fd0ebbf87090358644b62f7dfa0d8dcdcad9c5dc322ce28eaac  ./wxcode-modal/spot_ties/20201209T1400Z-weather_symbols-PT01H.nc
-c798e3530eac9a45305711d5e91daa56d5005b826ce85bb48e2889e328ea965c  ./wxcode-modal/spot_ties/20201209T1500Z-weather_symbols-PT01H.nc
-83ba8460875232fc555976872cdb5ac59d5092ccc0da88f88beb9168a3403ac3  ./wxcode-modal/spot_ties/20201209T1600Z-weather_symbols-PT01H.nc
-a6698b2aae8bc223a2d362ccb3047dd9882e6c0df21962e55c577e1ad5850e29  ./wxcode-modal/spot_ties/20201209T1700Z-weather_symbols-PT01H.nc
-9290b2899eedfca038226c6707e100025ce68b0798972a60360845ccf36858d2  ./wxcode-modal/spot_ties/20201209T1800Z-weather_symbols-PT01H.nc
-ceb7106e5b8919590327a75d4a8b740fb1a2c7cb5a271a5bf980f0c1396c7d22  ./wxcode-modal/spot_ties/kgo.nc
+f65618c59e938286feee2720bb3be6bf6b1d32b1e439319f9eeaa538cf605fcf  ./wxcode-modal/blend_mismatch_inputs/20201209T0700Z-weather_symbols-PT01H.nc
```

Comment:
It looks like the units for the `wmo_id` and `met_office_site_id` coordinates have changed in this acceptance test branch from `no_unit` to `unknown`. I think `no_unit` is preferred for these coordinates e.,g. [in build_spotdata_cube](https://github.com/metoppv/improver/blob/master/improver/spotdata/build_spotdata_cube.py#L120), so could this be corrected?